Extension/package.json

```diff
@@ -2364,6 +2364,42 @@
           "description": "%c_cpp.configuration.doxygen.generatedStyle.description%",
           "scope": "resource"
         },
+        "C_Cpp.doxygen.sectionTags": {
+          "type": "array",
+          "default": ["tparam","param", "returns", "retval", "exceptions", "deprecated", "warning", "note" ],
```

Comment:
Can you add "attention" to the defaults?

"result" and "return" are the same as "returns" -- should those be added too?

Reply:
I cannot add a flag for return in order to recognize it as a default the same way we do returns. In order to keep consistency rather than trying to find a workaround I just removed it since we already support "returns" which has the same functionality.

Reply:
I don't understand. Why exactly can't you add "return"? The user's code might use "return".